oak-blob/src/main/java/org/apache/jackrabbit/oak/spi/blob/AbstractBlobStore.java

```diff
@@ -119,6 +120,11 @@ public abstract class AbstractBlobStore implements GarbageCollectableBlobStore,
      */
     private int blockSize = 2 * 1024 * 1024;
 
+    /**
+     * Encode in Base 32, hex encoding, no line breaks, padding with "="
+     */
+    private static Base32 BASE32ENCODER = Base32.builder().setLineSeparator(new byte[0]).setPadding((byte)'=').get();
+
```

Comment:
Fehlt da nicht .setEncodeTable(...) ?

Reply:
It was.

I now changed to the old-style constructor, because we otherwise would have to speficy the actual values (the constants in the Base32 class are private).

Reply:
Yes, that's a design flaw in the builder API.